src/openrct2-ui/scripting/CustomMenu.cpp

```diff
@@ -20,15 +20,16 @@ namespace OpenRCT2::Scripting
     std::optional<CustomTool> ActiveCustomTool;
     std::vector<CustomToolbarMenuItem> CustomMenuItems;
 
-    static constexpr std::array<std::string_view, CURSOR_COUNT> CursorNames = {
+    static constexpr std::array<std::string_view, EnumValue(CursorID::Count)> CursorNames = {
         "arrow",         "blank",      "up_arrow",      "up_down_arrow", "hand_point", "zzz",         "diagonal_arrows",
         "picker",        "tree_down",  "fountain_down", "statue_down",   "bench_down", "cross_hair",  "bin_down",
         "lamppost_down", "fence_down", "flower_down",   "path_down",     "dig_down",   "water_down",  "house_down",
         "volcano_down",  "walk_down",  "paint_down",    "entrance_down", "hand_open",  "hand_closed",
     };
 
-    template<> DukValue ToDuk(duk_context* ctx, const CURSOR_ID& value)
+    template<> DukValue ToDuk(duk_context* ctx, const CursorID& cursorId)
     {
+        int8_t value = EnumValue(cursorId);
```

Comment:
Change the type to auto please, this avoids having type conflicts when the enum type changes.